docs/interactions/Application_Commands.md

```diff
@@ -1,83 +1,130 @@
+## Application Command Object
 
-Slash Commands set your users up for success instead of confusion and frustration. They separate how users think and how your code works, meaning no matter how complex your codebase and commands may get, people who love your bot will find it approachable and easy to use.
+> info
+> `CHAT_INPUT` command names must be all lowercase matching `^[\w-]{1,32}$`. `USER` and `MESSAGE` commands may be mixed case and can include spaces.
+
+###### Application Command Structure
 
-Let's get started!
+| Field               | Type                                                                                                                                           | Description                                                                | Valid Types |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- | ----------- |
+| id                  | snowflake                                                                                                                                      | unique id of the command                                                   | all         |
+| type?               | one of [application command type](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-types)                | the type of command, defaults `1` if not set                               | all         |
+| application_id      | snowflake                                                                                                                                      | unique id of the parent application                                        | all         |
+| guild_id?           | snowflake                                                                                                                                      | guild id of the command, if not global                                     | all         |
+| name                | string                                                                                                                                         | 1-32 character name                                                        | all         |
+| description?        | string                                                                                                                                         | 1-100 character description                                                | CHAT_INPUT  |
```

Comment:
```suggestion
| description         | string                                                                                                                                         | 1-100 character description for `CHAT_INPUT` commands, empty string for `USER` and `MESSAGE` commands                                               | all         |
```
this table will have to be reformatted anyway

Reply:
I'll have to think about this, since it's always returned in the payload you receive, but optional in the creation ><